src/IO/ADIOS/ADIOS2IOHandler.cpp

```diff
@@ -2616,17 +2679,28 @@ namespace detail
     {
         if (!m_engine)
         {
+            auto tempMode = m_mode;
             switch (m_mode)
             {
             case adios2::Mode::Append:
+#ifdef _WIN32
+                /*
+                 * On Windows, ADIOS2 Append mode only works with existing
```

Comment:
Let's mention the ADIOS2 version here and link an issue?

Reply:
```suggestion
                 * On Windows, ADIOS2 2.8 Append mode only works with existing
```

Reply:
I'm not sure on affected ADIOS2 versions. I can test again next week, and then report an issue.

Reply:
Done